v2/pkg/genruntime/configmaps/validation.go

```diff
@@ -39,6 +45,32 @@ func ValidateDestinations(destinations []*genruntime.ConfigMapDestination) (admi
 		locations.Add(pair)
 	}
 
+	if destinationExpressions == nil {
```

Comment:
I don't think you need this guard clause; it doesn't change the behaviour.

Reply:
Yes this is a holdover from when I had a different input type -- good catch